README.md

```diff
@@ -0,0 +1,41 @@
+
+# Decidim Documentation 
+
+This is the new repository for the [Decidim Documentation website](https://antora.org/):
+
+Is built with [Antora](https://antora.org/), although for this version we're going to try to not have multi-repositories, as it's a hassle to work with. The idea is to have all the documents living on this same repository. 
+
+This project is on active development and it's not on production yet.
+
+For working with it, you can do it several ways: 
+
+## Install locally 
+
+### Docker
```

Comment:
Also, I think manual installation should go first, and using specific tech later.

Reply:
Ok, that makes sense... I'm just trying to push "docker as the first option" on Decidim because is more consistent and cleaner, but maybe the world is not ready yet :thinking:

Reply:
Totally agree, running docker make installation easier. People who know docker will check it first, but a person who don't know docker will have to learn/install it first before running any docker command.

Making the user just copy/paste to get something working seems more interesting, even if it's longer